README.md

```diff
@@ -1,6 +1,39 @@
 # Google Cloud Platform OpenJDK Docker Image
 
-This repository contains the source for the `gcr.io/google_appengine/openjdk` [docker](https://docker.com) image. This image can be used as the base image for running Java applications on [Google App Engine Flexible Environment](https://cloud.google.com/appengine/docs/flexible/java/) and [Google Container Engine](https://cloud.google.com/container-engine).
+This repository contains the source for the `gcr.io/google-appengine/openjdk` [docker](https://docker.com) image. This image can be used as the base image for running Java applications on [Google App Engine Flexible Environment](https://cloud.google.com/appengine/docs/flexible/java/) and [Google Container Engine](https://cloud.google.com/container-engine).
+
+## App Engine Flexible Environment
+When using App Engine Flexible, you can use the runtime without worrying about Docker by specifying `runtime: java` in your `app.yaml`:
+```yaml
+runtime: java
+env: flex
+```
+The runtime image `gcr.io/google-appenine/openjdk` will be automatically selected if you are attempting to deploy a JAR (`*.jar` file).
+
+If you want to use the image as a base for a custom runtime, you can specify `runtime: custom` in your `app.yaml` and then
+write the Dockerfile like this:
+
+```dockerfile
+FROM gcr.io/google-appengine/openjdk
+ADD your-application.jar app.jar
```

Comment:
I couldn't help but notice that you used `ADD` here and `COPY` below in the GKE example. According to docker's [best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy) `COPY` is preferred.